xbmc/addons/kodi-dev-kit/include/kodi/c-api/addon-instance/pvr/pvr_defines.h

```diff
@@ -64,7 +64,7 @@ extern "C"
    */
   struct PVR_HANDLE_STRUCT
   {
-    void* callerAddress; /*!< address of the caller */
+    const void* callerAddress; /*!< address of the caller */
```

Comment:
It wouldn't hurt to bump the minor version

Reply:
@ksooo I meant patch version haha but no effective difference and maybe minor was the right choice because const is a narrowing operator and could break API.